docs/source/models/engine_args.rst

```diff
@@ -45,6 +45,7 @@ Below, you can find an explanation of every engine argument for vLLM:
     * "safetensors" will load the weights in the safetensors format.
     * "npcache" will load the weights in pytorch format and store a numpy cache to speed up the loading.
     * "dummy" will initialize the weights with random values, mainly for profiling.
+    * "tensorizer" will load serialized weights using `CoreWeave's Tensorizer model deserializer. <https://github.com/coreweave/tensorizer>`_.
```

Comment:
Can you add a link to the example doc here?

Reply:
Do you mean `examples/tensorize_vllm_model.py`? The tensorizer repo itself is the best resource for tensorizer documentation.

Reply:
yes!

Reply:
I think it is almost necessary to look at that file for any user who wants to use this feature. I think you should also mention tensorizer should be downloaded with pip install

Reply:
Added!